src/components/Header/index.js

```diff
@@ -67,6 +78,9 @@ const Header = ({ path, blue, logoColour }) => {
                   <HeaderAnchor activeClassName="active" to="/join-us/">
                     Join Us
                   </HeaderAnchor>
+                  <HeaderAnchor activeClassName="active" to="/join-us/">
+                    About Us
```

Comment:
@antoniocapelo I'm guessing that this was only in here for the purposes of development? i.e. checking what it looked like with an extra link?

We need to remove this, right? If the PR is going into `master`, rather than the `about-us-page` branch?

Reply:
@jsms90 yes, I just put it there for antonas to see how it will behave in the future, which he approved this morning :-) 

![image](https://user-images.githubusercontent.com/3236388/52956758-01c88c80-3388-11e9-8880-77b56c597188.png)

So if you approve it, I can remove that extra link and merge it :-)